README.md

```diff
@@ -54,14 +54,28 @@ To develop local extensions use the `docker-compose.dev.yml` file:
 
 To build the images:
 
-	docker-compose -f docker-compose.dev.yml build
+	use the script `./rebuild-ckan.sh`  # If starting from new, the script will take at least 10 minutes to run.
 
 To start the containers:
 
-	docker-compose -f docker-compose.dev.yml up
+	`docker-compose -f docker-compose.dev.yml up`
 
 See [CKAN Images](#ckan-images) for more details of what happens when using development mode.
 
+To ssh onto the ckan container:
+
+    `docker exec -it docker-ckan_ckan-dev_1 bash`
+
+To ssh onto the postgres container:
+
+docker exec -it ckan_db psql -U ckan
+
+export CKAN_INI=/srv/app/production.ini
+paster --plugin=ckan -c /srv/app/production.ini
```

Comment:
It might be good to put this in backticks so it's rendered as code.

Reply:
:+1: